packages/truffle-compile/test/test_supplier.js

```diff
@@ -1,10 +1,11 @@
 const debug = require("debug")("compile:test:test_supplier");
-const fs = require("fs");
+const fs = require("fs-extra");
```

Comment:
How do you feel about changing all instances of `fs` to `fse`? Cruz and I have been following this convention, I don't know if you would care to participate in the party...? If you don't feel like it, then I won't fight you :)

Reply:
you got it buddy